keanu-examples/bayesianMethodsForHackers/src/test/java/com/examples/ChallengerDisasterTest.java

```diff
@@ -3,12 +3,16 @@
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 import org.junit.Before;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.within;
 
 public class ChallengerDisasterTest {
 
+    private final Logger log = LoggerFactory.getLogger(ChallengerDisasterTest.class);
```

Comment:
The convention is always to make it `static`. However @gordoncaleb has just pointed out to me that you can annotate the class with Lombok's `@Slf4j` and then you can remove this line!